packages/use-contractkit/src/utils/metamask.ts

```diff
@@ -0,0 +1,233 @@
+import { ContractKit, newKit } from '@celo/contractkit';
+import { GoldTokenWrapper } from '@celo/contractkit/lib/wrappers/GoldTokenWrapper';
+import { StableTokenWrapper } from '@celo/contractkit/lib/wrappers/StableTokenWrapper';
+import Web3 from 'web3';
+
+import { Alfajores, Baklava, Mainnet } from '../constants';
+import { Ethereum } from '../global';
+import { ChainId, Network } from '../types';
+
+const CELO_PARAMS = Object.freeze({
+  chainName: 'Celo',
+  testnet: false,
+  nativeCurrency: {
+    name: 'CELO',
+  },
+});
+
+const ALFAJORES_PARAMS = Object.freeze({
+  chainName: 'Alfajores Testnet',
+  testnet: true,
+  nativeCurrency: {
+    name: 'A-CELO',
+  },
+});
+
+const BAKLAVA_PARAMS = Object.freeze({
+  chainName: 'Baklava Testnet',
+  testnet: true,
+  nativeCurrency: {
+    name: 'B-CELO',
+  },
+});
+
+const params: { [chain in ChainId]: typeof CELO_PARAMS } = {
+  [ChainId.Mainnet]: CELO_PARAMS,
+  [ChainId.Alfajores]: ALFAJORES_PARAMS,
+  [ChainId.Baklava]: BAKLAVA_PARAMS,
+};
+
+interface ERC20Token {
+  address: string;
+  name: string;
+  symbol: string;
+  decimals: number;
+  image?: string;
+}
+
+interface AddERC20TokenParameter {
+  type: 'ERC20';
+  options: ERC20Token;
+}
+
+interface CeloTokens {
+  CELO: GoldTokenWrapper;
+  cUSD: StableTokenWrapper;
+  cEUR: StableTokenWrapper;
+}
+type StableTokens = Omit<CeloTokens, 'CELO'>;
```

Comment:
There has to be a place for it so I don't need to re-invent these, but didn't find it. Any pointers @dckesler ?